utils/text_formatting.tsx

```diff
@@ -262,6 +263,16 @@ export function formatText(
         output = `<span class="all-emoji">${output.trim()}</span>`;
     }
 
+    if (postId) {
+        // unwrap the output from an enclosing p tag and add a span that will serve as a
+        // palceholder for `messageToHtmlComponent` function later on
+        if (output.startsWith('<p>') && output.endsWith('</p>')) {
+            output = `${output.slice(0, -4)}<span id='postEdited_${postId}'>(edited)</span></p>`;
```

Comment:
Would we be using this ID for anything else? I wonder if using something like `data-edited-id=${postId}` would be a bit better so that we're not adding extra meaning to an ID attribute that might be used elsewhere

Reply:
either way should be fine... I will adjust that using the `dataset`